api/server/structs/endpoints_beacon.go

```diff
@@ -201,7 +201,7 @@ type DepositSnapshot struct {
 
 type GetIndividualVotesRequest struct {
 	Epoch      primitives.Epoch            `json:"epoch"`
-	PublicKeys [][]byte                    `json:"public_keys,omitempty"`
+	PublicKeys [][]string                  `json:"public_keys,omitempty"`
```

Comment:
This is incorrect, it should be `[]string` because a string is a slice of bytes, not a single byte